homeassistant/components/light/lutron.py

```diff
@@ -0,0 +1,99 @@
+"""Support for Lutron lights."""
+import asyncio
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.components.lutron import (
+    LutronDevice, LUTRON_DEVICES, LUTRON_GROUPS, LUTRON_CONTROLLER)
+
+DEPENDENCIES = ['lutron']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
```

Comment:
Yeah, if you library will not update entity immediately, you can use `add_devices` with `True`, I add this for this case.

You can set hass like other entity does who need to have hass. You can also listen to HOMEASSISTANT_START Event and start your `._controller.subscribe` after this event. Or you check if hass is not None bevor process a state update.